modules/media/help/media.md

```diff
@@ -0,0 +1,17 @@
+# Media
+
+The Media module serves as a repository for files associated with a particular candidate-timepoint or instrument in a study.  Users can access and upload files such as PDF scans, recordings, log files, or stimulus presentation media in formats such as .pdf, .mp4, .mp3, and .txt. 
+
+## Searching for a Media File

```

Comment:
This (and the file above) are inconsistent in if there's a space under the header or not. Sometimes there's an empty line, sometimes there's not.